task-manager/src/components/TodaysTasks.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <v-card elevation="2" outlined>
+        <v-card-title class="headline">
+            <v-badge
+                :content="todaysTasks.length || '0'"
+                color="primary"
+                floating
+                >Today's Tasks</v-badge
+            >
+        </v-card-title>
+        <v-divider></v-divider>
+
+        <ag-grid-vue
```

Comment:
can we put all the stuffs related to ag grid in a separate component the pass the codefs , rowdata ... from parent rest all ag grid data should be in the child component

this component should be reusable in all files

Doing this, we'll not have to repeat the same code multiple times

Reply:
Okay, I will make it.